docs/upgrade/0.3.5-to-0.3.6.rst

```diff
@@ -91,11 +91,16 @@ Greeter*.
      git stash pop
 
 #. Make sure you have Ansible installed. If running ``which ansible`` returns 
-   ``ansible not found``, you should ::
+   nothing, you should ::
```

Comment:
@garrettr I don't think so. I ran `which` on Tom's admin workstation which uses Tails 1.x, and I ran version 2.4 on a VirtualBox VM on my computer, and saw the same behavior.